multigeiger/thp_sensor.cpp

```diff
@@ -4,29 +4,61 @@
 
 #include <Adafruit_Sensor.h>
 #include <Adafruit_BME280.h>
+#include <Adafruit_BME680.h>
 
 #include "log.h"
 #include "thp_sensor.h"
 
-static bool have_thp = false;
+static bool have_thp;
+static int type_thp = 0;
```

Comment:
maybe one could do it with just one of them. 0 == no sensor, 280 = bme280, 680 == bme680.

Reply:
will do.